tests/unit/DetailComment.spec.ts

```diff
@@ -42,49 +39,40 @@ describe('DetailComment', () => {
     expect(vm.value).toBe('Initial comment')
     expect(vm.placeholder).toBe('Enter Comment Here')
     expect(vm.autofocus).toBe(true)
-
-    wrapper.destroy()
   })
 
   // FUTURE: fix this test
-  it.skip('emits valid event when value prop is changed', async () => {
-    const wrapper = shallowMount(DetailComment,
+  it('emits valid event when value prop is changed', async () => {
+    const wrapper = mount(DetailComment,
       {
+        props: {
+          value: 'testing 1 2 3'
+        },
         vuetify
       })
 
-    // change the value
-    // NB: need to wait for debounce
-    wrapper.setProps({ value: 'testing 1 2 3' })
     await sleep(300)
     await flushPromises()
 
     // verify valid event
     expect(wrapper.emitted('valid').pop()[0]).toEqual(true)
-
-    wrapper.destroy()
   })
 
   it('emits events when value model is changed', async () => {
-    const wrapper = shallowMount(DetailComment,
+    const wrapper = mount(DetailComment,
       {
+        props: {
+          value: 'testing 1 2 3'
+        },
         vuetify
       })
     const vm: any = wrapper.vm
 
-    // const element = wrapper.find('#detail-comment-textarea')
-    // element.setValue('testing 4 5 6')
-
-    // apparently you can't set a textarea's value, so do it explicitly
-    // NB: need to wait for debounce
-    vm.onValueChanged('testing 4 5 6')
-    await sleep(300)
-    vm.emitInput('testing 4 5 6')
+    const element = wrapper.find('#detail-comment-textarea')
+    element.setValue('testing 4 5 6')
 
     // verify valid and input events
     expect(wrapper.emitted('valid').pop()[0]).toEqual(true)
-    expect(wrapper.emitted('input').pop()).toEqual(['testing 4 5 6'])
-
-    wrapper.destroy()
+    // expect(wrapper.emitted('updateValue')[0]).toEqual(['testing 4 5 6'])
```

Comment:
Couldn't get this working but will keep looking for solution.......

Reply:
You may have to trigger an update on the textarea (submit or blur maybe?), to get it to recognize the new value you set. Do a global search (maybe in other UIs, too) for `setValue` and see what was done.

Reply:
I haven't been able to resolve this and looking into it. I would like to merge this PR first so that other developers can start updating unit tests for the components they are working on. I will keep a note of this and work on this unit test later .

Reply:
@ketaki-deodhar Not sure if you've found this already, but this comment from filings UI might help. Apparently you can't set a textarea's value so you might have to do it explicitly: https://github.com/bcgov/business-filings-ui/blob/main/tests/unit/DetailComment.spec.ts#L75-L82

Reply:
Sounds good to me.

Don't spend too long (don't get stuck) on anything. For now we just need to make things like-for-like in Vue3. Thanks.